### PR TITLE
Update README to reflect new features

### PR DESCRIPTION
**Pull request by rquander**
Update README to reflect new features recently added to the repository.

Requested review from @sathnaga to ensure changes are accurate.

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -220,6 +225,27 @@
...
         IMPORTANT: Complete all Guest VM Install prerequisites listed in the Prerequisites
                    section above BEFORE running this script with the "--install" option.
+                   
+      
+      --input-file INPUTFILE
+        Use this option to provide your own custom input file to be used for host tests. 
+        Make sure the input file has the same structure and syntax as the config/tests/host/sanity.cfg 
+        file already in this repository.  This option is helpful if you wish to quickly reference 
+        your own host tests config file without editing the default config/tests/host/sanity.cfg file.
+
```
Comment (sathnaga) [inline README]: @rquander I would like to get inputs from @narasimhan-v on this

File: README [hunk 2]
```diff
@@ -245,7 +275,12 @@
          - The user adds a line "avocado-misc-tests/generic/ras.py" (without the quotes) to the end of the config/tests/host/sanity.cfg 
          file in this tests repository and saves the file.
          - The user runs the host_sanity suite using the avocado-setup.py script explained in Section 4 above.
-
+      
+      OPTION #2: If you want to run your own unique list of host tests without changing the config/tests/host/sanity.cfg file,
+      create your own config file based on the structure & syntax of the sanity.cfg file and then reference that specific file 
+      when you run avocado-setup.py using the "--input-file" option explained above.
```
Comment (sathnaga) [inline README]: Usage of input file needs to explained bit @narasimhan-v, could you pls help on thi

File: README (hunk 1 above)
Comment (narasimhan-v) [reply README]: @rquander There is a confusion between input file and config file here. I can provide a sample input file and also update this and other relevant sections of README in a separate commit.

File: README (hunk 2 above)
Comment (narasimhan-v) [reply README]: @sathnaga Sure.
@rquander As stated above, this is misguiding and is a sensitive section. I can provide a detailed explanation and example.

**Comment by narasimhan-v** [comment]
@rquander Please merge this PR without input-file section, and I can send a PR on top of that.
@sathnaga Please advise if that is fine.

**Comment by rquander** [comment]
I remove the --input-file section of the updated README.  Merging now.  Will look for the new PR from @narasimhan-v for a more detailed explanation of that option.

**Pull request merged**